### PR TITLE
780 clean dual simulations summary

### LINKED ISSUE #780: `DualSimulationsSummary` @pengguanya

### PR DESCRIPTION
**Pull request by pengguanya**
# Pull Request

Fixes #780

### CODE REVIEW

File: R/Simulations-class.R [hunk 1]
```diff
@@ -388,44 +388,38 @@ DualSimulations <- function(rho_est,
...
-##'
-##' @export
-##' @keywords classes
+# class ----
+
+#' `DualSimulationsSummary`
+#'
+#' @description `r lifecycle::badge("stable")`
+#' This class captures the summary of dual-endpoint simulations output.
+#' In comparison to parent class [`SimulationsSummary`], it has additional slots.
```
Comment (danielinteractive) [inline R/Simulations-class.R:399]:
```suggestion
#' In comparison to its parent class [`SimulationsSummary`], it has additional slots.
```

**Review by danielinteractive — COMMENTED** [review]
Thanks @pengguanya , looks good so far, not sure if you need to adjust downstream methods because of the slot renaming?

File: R/Simulations-class.R (hunk 1 above)
Comment (pengguanya) [reply R/Simulations-class.R:399]: I have adjusted a few methods like plot and summary in R/Simulations-methods.R and tested to see there is no error anymore in the example.Rmd and few other vignettes. Do you think there are other places I should look at?

**Comment by pengguanya** [comment]
> Thanks @pengguanya , looks good so far, not sure if you need to adjust downstream methods because of the slot renaming?

@danielinteractive all done. please review and approve

**Pull request merged**